next.config.js

```diff
@@ -1,5 +1,6 @@
-// next.config.js
-const withMDX = require('@zeit/next-mdx')()
+const withMDX = require('@next/mdx')()
+
 module.exports = withMDX({
-  target: 'serverless'
+  target: 'serverless',
+  pageExtensions: ['jsx', 'js', 'ts', 'tsx', 'mdx']
```

Comment:
```suggestion
  pageExtensions: ['js', 'mdx']
```

Reply:
Makes sense, but any special reason to not keep the other ones to?

Reply:
Few `ms` faster build times, because it's not checking all possible extensions.

Reply:
Good enough